build.py

```diff
@@ -733,34 +740,44 @@ def dockerfile_prepare_container_linux(argmap, backends):
 
 # Common dependencies. FIXME (can any of these be conditional? For
 # example libcurl only needed for GCS?)
+# libgomp1 is used for Onnxruntime
```

Comment:
Should this be conditioned on whether "onnxruntime" is in `backends`?

Reply:
I wasn't sure if it was worth adding an extra RUN command in the layer. I think the GPU min container installs it by default

Reply:
Can you elaborate on the worry of adding `RUN` instructions? Regardless of that, there is actually ways to resolve the backend required package at the Python script level:
```
RUN apt-get update && \
    apt-get install -y --no-install-recommends \
            libb64-0d \
            libcurl4-openssl-dev \
            libre2-5 \
            git \
            dirmngr \
            libnuma-dev \
            curl \
            {ort_dependencies} && \
    rm -rf /var/lib/apt/lists/*
'''.format(ort_dependencies=ort_dependencies)
```
And you prepare the `ort_dependencies` variable earlier to be something like:
```
ort_dependencies = "libgomp1" if "onnxruntime" in backneds else ""
```

Reply:
Run adds additional layers, which is discouraged in dockerfiles I think. 
ref: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Reply:
Each layer in docker adds some overhead... but I agree with Guan in the best way to address conditionally adding libgomp1 that does not require an extra RUN.